src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/SettingsPanel.form

```diff
@@ -223,6 +217,35 @@
                   </component>
                 </children>
               </grid>
+              <grid id="46307" layout-manager="GridLayoutManager" row-count="1" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+                <margin top="0" left="0" bottom="0" right="0"/>
+                <constraints>
+                  <grid row="8" column="0" row-span="1" col-span="1" vsize-policy="3" hsize-policy="3" anchor="0" fill="3" indent="0" use-parent-layout="false"/>
+                </constraints>
+                <properties/>
+                <border type="none"/>
+                <children>
+                  <component id="65f1d" class="javax.swing.JLabel">
+                    <constraints>
+                      <grid row="0" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+                    </constraints>
+                    <properties>
+                      <text value="Clone Base URL"/>
+                      <toolTipText value="Override clone base URL if it differs from gerrit default"/>
+                    </properties>
+                  </component>
+                  <component id="7a077" class="javax.swing.JTextField" binding="cloneBaseUrlTextField">
+                    <constraints>
+                      <grid row="0" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="6" anchor="8" fill="1" indent="0" use-parent-layout="false">
+                        <preferred-size width="150" height="-1"/>
+                      </grid>
+                    </constraints>
+                    <properties>
+                      <toolTipText value="Override clone base URL if it differs from gerrit default"/>
```

Comment:
This looks duplicated. Is it required for both label and input? Probably you could also add this text as label below this input which would be better visible.